backend/test/config.spec.js

```diff
@@ -74,8 +74,18 @@ describe('config', function () {
         SESSION_SECRET: 'secret'
       }
 
+      let existsSyncSpy
+      let readFileSyncSpy
+
       beforeEach(() => {
+        existsSyncSpy = jest.spyOn(fs, 'existsSync')
```

Comment:
I would avoid mocking the filesystem sicne it is used to run the tests. This could lead to strange unpredictable situations. It is better to use a local wrapper and mock the wrapper.
```js
const fs = require('fs')
const readFileSync = fs.readFileSync
// or
const readTextFile = filename => fs.readFileSync(filename, 'utf8')
```

Reply:
as just discussed in person there other tests also mock the fs directly without issues and we leave it as is. Maybe it was an issue in earlier jest versions?